projects/components/src/table/table.component.ts

```diff
@@ -559,6 +610,33 @@ export class TableComponent
     return !!this.data && !!this.columnConfigs && (this.pageable ? !!this.paginator : true);
   }
 
+  public showColumnResizeDivider = (columns: TableColumnConfig[], index: number): boolean => {
+    if (index === 0) {
+      return false;
+    }
```

Comment:
Nitpick: May as well return false for any value greater than or equal to zero.

Reply:
I didn't get you, why?

The usage is this function to show the resizable column divider, which we want to show by default apart from
1. When it is a first column
2. When the previous column was a state column

Reply:
> any value greater than or equal to zero.

You said `>= 0`, did you mean `<= 0`

Reply:
Oh yeah, my bad!